core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -147,6 +147,7 @@ class KafkaServer(
 
   var kafkaScheduler: KafkaScheduler = _
 
+  var kraftControllerNodes: Seq[Node] = Seq.empty
```

Comment:
Let's just initialize this to `_` like the other class members here. It's not pretty, but it's better to be consistent